slick/src/sphinx/gettingstarted.rst

```diff
@@ -33,7 +33,8 @@ following to your build definition - ``build.sbt`` or ``project/Build.scala``:
 .. parsed-literal::
   libraryDependencies ++= Seq(
     "com.typesafe.slick" %% "slick" % "|release|",
-    "org.slf4j" % "slf4j-nop" % "1.6.4"
+    "org.slf4j" % "slf4j-nop" % "1.6.4",
+    "com.typesafe.slick" %% "slick-hikaricp" % "3.1.1"
```

Comment:
Just tried it locally to check the version injection in sphinx, and I get `"com.typesafe.slick" %% "slick" % "3.2.0-SNAPSHOT"`.   Using `|release|` in the hikaricp line would be good!

If users are getting caught out on this one I think it's better to include the dependency it by default that not.  An reasonable alternative would be to include the dependency line as an example in the text where pooling is mentioned... but one way or another it should be here.